pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -144,6 +147,7 @@ let
 
     buildInputs = [
       jemalloc
+      (if mklSupport then mkl else null)
```

Comment:
While it works technically, I think adding `null` to the list is not very nice. Please add `++ lib.optionals mkSupport [mkl]` to the end of the list instead. (cc @matthewbauer)

Reply:
Definitely like that better as well.